apps/web/app/layout.tsx

```diff
@@ -3,14 +3,20 @@ import type { Metadata } from "next";
 import { ReactNode, JSX } from "react";
 // BC Gov
 import "@bcgov/bc-sans/css/BC_Sans.css";
-// packages
+// repo
+import Header from "@repo/ui/header";
+import {
+  ID_FOOTER,
+  ID_MAIN_CONTENT,
+  ID_MAIN_NAVIGATION,
+} from "@repo/constants/src/ids";
 import "@repo/ui/cssVariables";
 // local
 import { ClientProviders } from "./ClientProviders";
 import "./globals.css";
 
 export const metadata: Metadata = {
-  title: "BC Gov - Building Code - Walkthroughes",
+  title: "BC Gov - 2024 Building Code - Walkthroughes",
```

Comment:
Typo on Walkthroughs?

Reply:
Weird. My IDE was saying Walkthroughs was spelled wrong and wanted the es. Added to the dictionary and fixed across the board.